src/components/projects/projectId/lookup-lists/DeleteLookupListsModal.tsx

```diff
@@ -21,19 +20,11 @@ export default function DeleteLookupListsModal(props: DeleteLookupListsModalProp
 
     const [abortButton, setAbortButton] = useState<ModalButton>(ABORT_BUTTON);
 
-    const [deleteLookupListMut] = useMutation(DELETE_LOOKUP_LIST);
-
-
     const deleteLookupLists = useCallback(() => {
         checkedLookupLists.forEach((checked, index) => {
             if (checked) {
                 const lookupList = lookupLists[index];
-                deleteLookupListMut({
-                    variables: {
-                        projectId: projectId,
-                        knowledgeBaseId: lookupList.id
-                    }
-                }).then((res) => {
+                deleteKnowledgeBase(projectId, lookupList.id, (res) => {
```

Comment:
if there is time, delete and create Knowledgebase should be renamed to delete and create Lookuplist since we use those names now.

Reply:
yes i agree, i am resolving the issue since it's non prio